tests/test_api.py

```diff
@@ -4,6 +4,12 @@
 import json
 import os
 
+@pytest.fixture(scope="module")
```

Comment:
this works without the autouse ?
I though you needed either an autouse=True or to inject the fixture in another one. :thinking:

Reply:
Yes it works for each test individually and also all tests together.